.circleci/config.yml

```diff
@@ -158,36 +252,57 @@ jobs:
           command: |
             ./ci-tools/run-tests.sh
 
-  tests-1909:
-    <<: *TESTS
-    environment:
-      NIXPKGS_REV: "19.09"
-
-  tests-2009:
+  tests-2105:
     <<: *TESTS
     environment:
-      NIXPKGS_REV: "20.09"
+      NIXPKGS_REV: "21.05"
```

Comment:
I guess these are tags of the initial release of that branch, which is probably fine for running tests.